public/css/main.css

```diff
@@ -538,11 +538,11 @@ body.darkBG .progress {
   padding: .1rem .5rem;
   height: 25px;
 }
-.tx-col-l{
-  max-width:54%;
-}
-.tx-col-r{
-  max-width:46%;
+.tx-type-stub{
+  max-width: 80px;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow:hidden;
```

Comment:
Please see the discussion with gozart [here](https://github.com/decred/dcrdata/pull/718/files/ec2db4682fe91adac7f957aec171be7e76a7e37f#discussion_r229499754), and the issue [here](https://github.com/decred/dcrdata/issues/782).  I don't pretend to be a competent frontend developers, so I almost always defer to gozart or whoever is working on a new design.  But we've gotten to a point where we need to be very careful and communicate all changes to main.css so as to keep it stable.